composer.json

```diff
@@ -32,7 +32,9 @@
         "composer/composer": "^1.9 || ^2.0",
         "egulias/email-validator": "^2.1 || ^3.0",
         "typo3fluid/fluid": "~2.5.11 || ^2.6.10",
-        "guzzlehttp/psr7": "^1.4.1, !=1.8.0",
+        "guzzlehttp/psr7": "^1.7, !=1.8.0",
+        "neos/fusion-afx": "*",
```

Comment:
That comes from the kickstarter, where it should only be an optional dependency (but that has not been tackled yet). That new requirement leads to the Fusion tests being run which yield the test failure.

> 1) Neos\Fusion\Tests\Unit\Service\HtmlAugmenterTest::invalidAttributesTests with data set #0 ('', array(array()), null, null, '<root>array value ignored</root>')
PHPUnit\Framework\Exception: Class Neos\Neos\Exception does not exist

So seems like Fusion still depends a bit too much on the neos core - so either we fix that, or we remove the dependency to fusion-X from here

Reply:
- Imagine the collection manifest would be updated automagically–then such a change would be impossible
- If the kickstarter requires it, it must be installed, simply removing the dependency sounds foolish

But may I ask: Why does the kickstarter have that dependency? I can't seem to find it using Fusion…

Reply:
> Why does the kickstarter have that dependency? I can't seem to find it using Fusion…

You can generate Fusion templates with it if you want to, which would mean you'd have to have Fusion installed. So that dependency should rather be a "suggests"?

Reply:
No, in that case the generated package must have a dependency on Fusion. Right?

Reply:
That is... correct. I guess I was thinking in terms of "the new package should then directly work without another composer update" or s.th. like that

Reply:
Probably. But with the use of `DistributionPackages` an update is needed / done after kickstarting a package anyway, no?

Reply:
Right - I was not thinking this fully through I guess and just sticked to how the Fluid dependency was handled, though that is indeed a direct dependency of the kickstarter.

We'd need to adjust https://github.com/neos/flow-development-collection/blob/master/Neos.Kickstarter/Classes/Command/KickstartCommandController.php#L119 to include the dependency for fusion-afx/form (and also fluid btw) depending on if generate-fusion is selected.